superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -753,6 +753,11 @@ export function useDatabaseValidation() {
         .catch(e => {
           if (typeof e.json === 'function') {
             return e.json().then(({ errors = [] }: JsonObject) => {
+              if (database?.parameters?.ssh) {
```

Comment:
I don't think this belongs here... if we don't want to show validation errors for some reason, then we shouldn't be running the validation API calls.